SVProgressHUD.xcodeproj/project.pbxproj

```diff
@@ -329,6 +329,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.3;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				MTL_ENABLE_DEBUG_INFO = NO;
+				PRODUCT_BUNDLE_IDENTIFIER = "com.transitapp.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
At the time of this 22cce21 com.transitapp was valid. However as we moved away from the transitapp group to an own group this should not be mentioned any more.